website/blog/2023-02-03-product-update.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "February 2023 dbt Update"
+description: "The latest product updates from dbt Labs"
+slug: dbt-product-update-2023-february
+# authors: [lauren_craigie] 
```

Comment:
Can you either add yourself as an author or uncomment Lauren here? Always nice to give a face to blog content.